### DISCUSSION TITLE
Touchscreen Lightgun support help please

### DISCUSSION THREAD

**Discussion by Jonathan7299** [flyinghead/flycast#606] [Q&A]
In the last feature request I posted, I was trying to ask if you guys could get the Touchscreen Lightgun to function like it does on the MAME core (wherever you touch on the game screen is where it will fire). If the Flycast core on Android already does that, can you please tell me how to set it up that way?

**Comment by flyinghead** [comment]
See https://github.com/flyinghead/flycast/issues/1446
